handlers/handlers.go

```diff
@@ -0,0 +1,67 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/Daniorocket/cookit-backend/lib"
+	"github.com/gorilla/mux"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type Handler struct {
+	Client       *mongo.Client
+	DatabaseName string
+}
+type JWT struct {
+	Token    string
+	Username string
+}
+type Pagination struct {
+	Data          interface{} `json:"data"`
+	Limit         string      `json:"limit"`
+	Page          string      `json:"page"`
+	TotalElements int64       `json:"totalElements"`
+}
+type ApiResponse struct {
+	Data   interface{} `json:"data"`
+	Status string      `json:"status"`
+	Code   int         `json:"code"`
+	Error  string      `json:"error"`
+}
```

Comment:
Nie sprawdzałem, ale wygląda na to że one chyba powinny być prywatne a nie publiczne?

Reply:
Zmieniam na prywatne, czy w takim razie struktura Handler również powinna być prywatna?